app/experimenter/nimbus-ui/src/lib/test-utils.tsx

```diff
@@ -116,3 +117,16 @@ export const assertSerializerMessages = async (
     }
   }
 };
+
+export const CurrentLocation = () => {
+  const location = useLocation();
+  return (
+    <p className="p-3">
+      Location:{" "}
+      <code data-testid="location">
+        {location.pathname}
```

Comment:
Looks like this is always rendering `/slug/edit`

Reply:
Yeah, that's just the canned value from mocks. Threw it in there just to make sure the path didn't get altered by the query parameter manipulation